cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -198,3 +201,50 @@ def test_get_dictionary_mapping_curie(self):
         ids = ["CL:NO_TERM"]
         with self.assertRaises(ValueError):
             self.writer._get_mapping_dict_curie(ids, curie_constraints)
+
+
+class TestSeuratConvertibility(unittest.TestCase):
```

Comment:
We have tried to keep the construction of AnnData objects in a separate fixture file, you can add to it or create a new one.  Probably the latter given how big the matrix you are creating is.

`./cellxgene_schema_cli/tests/fixtures/`

And most of the unittests here test for the the actual warning/error. Could you refractor for that?

Reply:
> And most of the unittests here test for the the actual warning/error. Could you refractor for that?

Do you mean I should test for the warning in each test?

Reply:
it would be sufficient to only to it under `test_add_warning_for_seurat_nonconvertible`